app/src/main/java/ca/chronofit/chrono/circuit/CircuitDashboardFrag.kt

```diff
@@ -157,6 +157,20 @@ class CircuitDashboardFrag : Fragment() {
     }
 
     private fun observeSettings() {
+        // Retrieve Settings if they exist
+        if (PreferenceManager.get<Int>("readyTime") != null) {
+            getReadyTime = PreferenceManager.get<Int>("readyTime")!!
```

Comment:
Add these as string consts. Either in res or in one of these class as a static member.

Reply:
ye done